data/basics/teal.go

```diff
@@ -134,6 +134,8 @@ type EvalDelta struct {
 	// When decoding EvalDeltas, the integer key represents an offset into
 	// [txn.Sender, txn.Accounts[0], txn.Accounts[1], ...]
 	LocalDeltas map[uint64]StateDelta `codec:"ld,allocbound=config.MaxEvalDeltaAccounts"`
+
+	Log []string `codec:"lg,allocbound=-"`
```

Comment:
Shouldn't `allocbound` be set to `logic.MaxLogCalls`? Maybe it makes sense to move the `MaxLogCalls` constant to the `config` package?

Reply:
how about leaving logic.MaxLogCalls as is but create a variable in config for decoding purposes.

Reply:
or should MaxLogCalls and MaxLogSize be consensus params? @jannotti

Reply:
I moved to MaxLogCalls to config. and allocbound is now set to config.MaxLogCalls.